doc/classes/TextEdit.xml

```diff
@@ -1676,6 +1676,9 @@
 			The size of the text outline.
 			[b]Note:[/b] If using a font with [member FontFile.multichannel_signed_distance_field] enabled, its [member FontFile.msdf_pixel_range] must be set to at least [i]twice[/i] the value of [theme_item outline_size] for outline rendering to look correct. Otherwise, the outline may appear to be cut off earlier than intended.
 		</theme_item>
+		<theme_item name="wrap_right_offset" data_type="constant" type="int" default="10">
+			Sets an additional right margin for line wrapping width.
```

Comment:
The name probably shouldn't use "right" since it is on the left if the layout is set to rtl.